src/components/utils/isDesktop.js

```diff
@@ -1,3 +1,3 @@
-const width = (window.matchMedia("(min-width: 800px)").matches)
```

Comment:
I think I understand your point now. I actually removed this file at one point when implementing the changes, but then I realised there were other files using the isDesktop module, so it actually made sense to keep it. Also in the future I might find better code for checking `isDocument`, because DetectRTC's goal is not focused on that feature, and from inspecting the code it actually uses user agents so it might not be so robust.